### PR TITLE
Climatic > Transform > Water Balance - Evaporation column 

### PR DESCRIPTION
**Pull request by bethanclarke**
Evaporation receiver R-code implemented and ready for testing. 

There is an issue with TestOkEnabled, it is only enabled for the Variable rdo if you click on Value rdo button and click Varable rdo again. Unfortunately I haven't been ale to fix this, so any help and suggestions would be great, thank you.

### CODE REVIEW

**Comment by shadrackkibet** [comment]
@bethanclarke is this ready now?

File: instat/dlgTransformClimatic.vb [hunk 1]
```diff
@@ -361,6 +380,10 @@ Public Class dlgTransformClimatic
         End If
     End Sub
 
+    Private Sub lblYear_Click(sender As Object, e As EventArgs) Handles lblYear.Click
```
Comment (shadrackkibet) [inline instat/dlgTransformClimatic.vb]: I this is not needed.

File: instat/dlgTransformClimatic.vb [hunk 2]
```diff
@@ -420,7 +443,27 @@ Public Class dlgTransformClimatic
...
+    Private Sub CoreControls_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverDate.ControlContentsChanged, ucrReceiverYear.ControlContentsChanged, ucrReceiverData.ControlContentsChanged, ucrNudSumOver.ControlContentsChanged, ucrNudCountOver.ControlContentsChanged, ucrInputSpellLower.ControlContentsChanged, ucrInputSpellUpper.ControlContentsChanged, ucrInputCondition.ControlContentsChanged, ucrInputColName.ControlContentsChanged, ucrNudWBCapacity.ControlContentsChanged
+        TestOkEnabled()
+    End Sub
+
+    Private Sub ucrPnlEvap_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrPnlEvap.ControlContentsChanged
+        Evaporation()
```
Comment (shadrackkibet) [inline instat/dlgTransformClimatic.vb]: You already have this sub in `valuechanged` event. So you don't need this here.

File: instat/dlgTransformClimatic.vb [hunk 3]
```diff
@@ -14,6 +14,7 @@
 ' You should have received a copy of the GNU General Public License 
 ' along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+Imports instat
```
Comment (shadrackkibet) [inline instat/dlgTransformClimatic.vb]: should be removed.

**Comment by shadrackkibet** [comment]
@bethanclarke  there is an issue with your design i can't test this.
image

**Comment by bethanclarke** [comment]
The issue above has been fixed and this is ready for testing @shadrackkibet.

File: instat/dlgTransformClimatic.vb [hunk 4]
```diff
@@ -462,8 +462,13 @@ Public Class dlgTransformClimatic
         End If
     End Sub
 
-    Private Sub ucrPnlEvap_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlEvap.ControlValueChanged
+    Private Sub ucrReceiverEvap_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrReceiverEvap.ControlValueChanged
+        TestOkEnabled()
```
Comment (shadrackkibet) [inline instat/dlgTransformClimatic.vb]: This works, but for consistency this should be on `contentschanged` event.

Comment (bethanclarke) [reply instat/dlgTransformClimatic.vb]: This is contentschanged on my file, have you got the most recent file changes?

**Comment by shadrackkibet** [comment]
Only one small issue to fix and this should be ready. I think the focus should be on Evaporation receiver when Variable radio button is checked.

**Comment by shadrackkibet** [comment]
@bethanclarke is this done?

**Comment by shadrackkibet** [comment]
@bethanclarke ?

**Comment by bethanclarke** [comment]
These changes have been included on another pull request. So I will close this one

**Pull request closed without merging**